crawler.go

```diff
@@ -173,6 +173,28 @@ func (c crawler) selecionaMesAno(ctx context.Context, tipo string) error {
 	year := fmt.Sprintf("//*[@title='%s']", c.year)
 	month := fmt.Sprintf("//*[@title='%s']", c.month)
 
+	selectedMonth, err := c.getSelectedMonth(ctx, tipo)
+	if err != nil {
+		log.Fatalf("erro ao obter mês selecionado no site: %v", err)
+	}
+
+	if selectedMonth == c.month {
+		return chromedp.Run(ctx,
+			// Seleciona ano
+			chromedp.Click(selectYear, chromedp.BySearch, chromedp.NodeReady),
+			chromedp.Sleep(c.timeBetweenSteps),
+
+			chromedp.Click(year, chromedp.BySearch, chromedp.NodeReady),
+			chromedp.Sleep(c.timeBetweenSteps),
+
+			chromedp.Click(selectYear, chromedp.BySearch, chromedp.NodeReady),
+			chromedp.Sleep(c.timeBetweenSteps),
+
+			chromedp.Click(year, chromedp.BySearch, chromedp.NodeReady),
+			chromedp.Sleep(c.timeBetweenSteps),
+		)
+	}
```

Comment:
O que fazer quando o mês selecionado não for? Temos que ter uma mensagem clara aqui.